### ISSUE TITLE
Memory leak?

### ISSUE THREAD

**Issue by siewers** [leighlondon/eml#1]
I've just installed this extension, and immediately the memory usage of VSCode went through the roof.
The extension does not work at all.

It _might_ be related to the insiders build of VSCode.

Version 1.21.0-insider
Commit efa33330fb0ab23b022bdff125311bdb21494b71
Date 2018-02-21T09:22:14.399Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64

**Comment by leighlondon** [comment]
Hmm, interesting. Had you opened an `eml` file to trigger the syntax highlighting or was that purely after installing the extension?

I'm on 1.20.1 but I'll check with the insiders build as well.

**Comment by siewers** [comment]
I opened an eml file, and it went nuts :)

**Comment by leighlondon** [comment]
Ah, that's not good. The memory usage on my Insiders build went up a bit, but not through the roof.
I played a bit fast and loose with some of the matchers so I'll amend them to see if I can get it down.

**Comment by siewers** [comment]
I'll make a small video of what's happening on my side.

**Comment by siewers** [comment]
Here's a short example (sorry for the awful sharing site, but the tool itself is pretty great :)
https://screenpresso.com/=J6Z5g

**Comment by leighlondon** [comment]
Thanks for that video! I've been able to reproduce the exact same problem by using a Windows machine as well (and it crashed the whole machine) so I'll be trying to fix that up ASAP.

**Comment by leighlondon** [comment]
An interim release has been published while I fix the issue (0.1.4).

**Comment by leighlondon** [comment]
This should now be fixed in a82f0d1 and version 0.2.1 has been published with the fix.

**tl;dr:** I accidentally added an extra regex op in development but didn't notice because it didn't impact macOS computers (but have fixed for everything now)

Thanks for letting me know! ✨

**Comment by siewers** [comment]
Interesting that it _didn't_ affect macOS... :) But thanks for fixing it... :)
